src/Eto/Forms/Controls/DocumentControl.cs

```diff
@@ -198,6 +217,107 @@ public bool AllowReordering
 		set { Handler.AllowReordering = value; }
 	}
 
+
+	/// <summary>
+	/// Gets or sets a value indicating the tabs can be navigated by next and previous buttons.
+	/// </summary>
+	/// <value><c>true</c> to allow navigating; otherwise, <c>false</c>.</value>
+	public bool AllowNavigating
```

Comment:
Would be nice to add the checkbox for playing with this property into Samples app.

Reply:
Applied your comment, you can have a look in this [commit](https://github.com/picoe/Eto/pull/2574/commits/771ed9ece1d1b7c38ed7c77e568df036043cb6b0)